### ISSUE TITLE
[Microsoft][SQL Server Native Client 11.0]Communication link failure

### ISSUE THREAD

**Issue by NoelAbrahams** [Azure/node-sqlserver#182]
Hi, 

After upgrading to Node 0.10.33 we're getting a bunch of `Communication link failure` errors.

Any idea how to fix this?

**Comment by meet-bhagdev** [comment]
It will only work with node 0.8.9(and some other 0.8.x) versions

**Comment by NoelAbrahams** [comment]
@meet-bhagdev, this actually works in 0.10.33. The error was to do with caching the communications object. You need to create it every time.

**Comment by meet-bhagdev** [comment]
@NoelAbrahams : Are you able to use npm directly on 0.10.33? I tried using npm install msnodesql and it won't work.

**Comment by NoelAbrahams** [comment]
I'm using the custom build - not npm directly. It was built some time ago and I haven't changed anything.

**Comment by meet-bhagdev** [comment]
Oh I see. Do you have Visual C++ 2010 for the gyp build?

**Comment by NoelAbrahams** [comment]
That was probably the version I had installed when I built it. Sorry, can't confirm as it was some time ago.

**Comment by meet-bhagdev** [comment]
I see! Thanks for the info. If you still use the driver and have feedback, do let me know. i am on the Microsoft team working on the next version and would love to see what you liked/did not like.

**Comment by NoelAbrahams** [comment]
Sure. It's good to see that it's going to be maintained. We have been using it for about a year and a half. The biggest pain point is not being able to use npm. Also every version should work reliably on Azure.

**Comment by meet-bhagdev** [comment]
Thanks for the feedback. I will pass it along :) As I said, npm will work but only for node 0.8.x. The next versions should not have any of these problems.

**Comment by NoelAbrahams** [comment]
@meet-bhagdev, 

I just installed VS 2015 RTM. The installation also installs node v0.12.2. My custom built node-sqlserver driver stopped working after that, and I had to uninstall node and install v0.10.33.

This is a problem because I believe VS 2015 requires the latest node in order to support some features (mobile development etc.).

I would really like to see an updated node-sqlserver driver on npm.
